Yosemite/Yosemite/Actions/MetaDataAction.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import Networking
+
+/// MetaDataAction: Defines all of the Actions supported by the MetaData.
+///
+public enum MetaDataAction: Action {
+
+    /// Update order metadata in the local database.
```

Comment:
nit: Sorry, from this comment, I don't clearly understand what the method is doing. I see that the implementation also calls the remote and not only updates the local database. 

The same applies to the following `updateProductMetaData`method as well.

Reply:
Now it should be more clear 👍 acc516e4ab78fb94f18b73c09df0052f5f57f6f8